external/grpc/extlib/zlib/trees.h

```diff
@@ -0,0 +1,128 @@
+/* header created automatically with -DGEN_TREES_H */
+
+local const ct_data static_ltree[L_CODES+2] = {
```

Comment:
add spaces

Reply:
it's external. so we don't need to follow the coding rule.